Exercise1.js

```diff
@@ -0,0 +1,6 @@
+function calculateAverage(numbers) {
+    let sumOfNumbers = numbers.reduce((acc, val) => acc + val,0)
```

Comment:
1. sum as a variable name is enough
2. No need of 0 as an initial value of the accumulator, by default it'll be the first element of the array